forgerock-auth/src/main/java/org/forgerock/android/auth/webauthn/WebAuthKeySelectionFragment.kt

```diff
@@ -38,17 +40,10 @@ class WebAuthKeySelectionFragment : DialogFragment() {
     private lateinit var binding: FragmentUserSelectBinding
 
     var continuation: CancellableContinuation<PublicKeyCredentialSource?>? = null
-        set(value) {
-            field = value
-            field?.invokeOnCancellation {
-                if (this.isVisible) {
-                    dismiss()
-                }
-            }
-        }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        isCancelable = false
```

Comment:
just a question . so if you set this , user cannot tap outside the dialog, they have to explicitly press the Cancel button. is that right ?